view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -137,7 +262,7 @@ define(
              * @return {boolean}
              */
             orderValueTooLow: function () {
-                return window.checkoutConfig.totalsData.base_grand_total < 3000;
```

Comment:
Magic numbers. Should be in config?

Reply:
this will come from capability api. that 3000 thb

Reply:
These are not in the new capabilities API, and there is no plan to put them there AFAIK

Reply:
depends how you look at it. vat in thailand is for particular merchant his capability, he has different VAT then in Japan.

Reply:
the same as above.. since even we were developing this functionality minimum has been changed from 5k to 3k.. there is chance it will be changed again. 
It needs to be in capabilities.